portfolio/src/main/webapp/script.js

```diff
@@ -3,10 +3,12 @@ let lastRandomIndex;
 /**
  *Fetch json practice, array of messages
  */
-function getData() {
-  fetch('/data').then(response => response.json()).then(messages => {
+function getData(maxLoad) {
+  console.log('got in here, maxload: ', maxLoad);
```

Comment:
Remove debug statement for code to check in

Reply:
okay, deleted them